gamenet/generate/loader.py

```diff
@@ -180,6 +182,24 @@ def fix_network(network, version):
                     NetClients("clients"),
                 ]),
             ]
+        if version == VERSION_0_7:
+            network.Connless += [
+                NetConnless("Info", "inf3", [
+                    NetU8("token"),
```

Comment:
Should be a `NetIntAny`, no? It seems to leak into the `version` string in your screenshot.

Reply:
![image](https://github.com/heinrich5991/libtw2/assets/20344300/1a51d718-ea92-4139-abff-fbd32fbcebc8)
yes looks much better :sweat_smile: